client/containers/DashboardOverview/collectionState.js

```diff
@@ -12,6 +12,9 @@ const findRankForSelection = (selections, index) =>
 	);
 
 const linkCollectionPubs = (overviewData, collection) => {
+	if (!collection) {
```

Comment:
I'm confused about how we could arrive in a state where the collection being passed to this component is not defined, but it seems like an upstream concern — if there's no collection in scope, we should not be calling the `useCollectionState()` hook.

Reply:
Yup - confusing indeed. Fixed with refactor in latest commit.